DaiQiOS/DaiQiOS/Todaychecklist.swift

```diff
@@ -1,41 +1,69 @@
 import SwiftUI
 
 struct TodayChecklist: View {
-    
-    var checklistItems: [ChecklistItem] = [
+    @State var checklistItems: [ChecklistItem] = [
         ChecklistItem(title: "Task 1", description: "Description 1"),
         ChecklistItem(title: "Task 2", description: "Description 2"),
         ChecklistItem(title: "Task 3", description: "Description 3")
     ]
-    
+
     var body: some View {
         NavigationView {
-            ZStack {
-                List(checklistItems) { item in
-                    TaskCell(checklistItem: item)
+            List {
+                Section {
+                    ForEach(checklistItems) { item in
```

Comment:
If we pass in `checklistItems.enumerated()` we will have the index inside of the closure

```suggestion
                    ForEach(checklistItems.enumerated()) { (index, item) in
```